### ISSUE TITLE
Fuzz test error: Failed to execute 'removeChild' on 'Node'

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/circuit-construction-kit-common#343]
Continuous testing is showing this accessibility related error:
```
circuit-construction-kit-dc : fuzz : built : run
Uncaught NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at e.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1114:27887)
    at e.removeSubtree (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1114:26627)
    at e.changedAccessibleContent (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1115:18225)
    at e.setAccessibleContent (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1093:18278)
    at e.set accessibleContent [as accessibleContent] (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1093:18377)
    at e.setAccessibleContentDisplayed (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21805)
    at e.setAccessibleContentDisplayed (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21813)
    at r.setAccessibleContentDisplayed (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21813)
    at e.setAccessibleContentDisplayed (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21813)
    at e.setAccessibleContentDisplayed (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21813)
Approximately 5/31/2017, 11:44:57 AM
circuit-construction-kit-dc : fuzz : built : run
Uncaught NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at e.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1114:27887)
    at e.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1114:27981)
    at e.removeSubtree (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1114:26627)
    at e.changedAccessibleContent (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1115:18225)
    at e.setAccessibleContent (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1093:18278)
    at e.set accessibleContent [as accessibleContent] (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1093:18377)
    at e.m [as invalidateAccessibleContent] (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:13313)
    at e.setAccessibleContentDisplayed (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21851)
    at e.set accessibleContentDisplayed [as accessibleContentDisplayed] (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1092:21921)
    at e.setAccessibleViewsHidden (https://bayes.colorado.edu/continuous-testing/snapshot-1496252697286/circuit-construction-kit-dc/build/circuit-construction-kit-dc_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1111:4491)
Approximately 5/31/2017, 11:44:57 AM
```

**Comment by samreid** [comment]
@jessegreenberg what is your estimate timeline for investigating this issue?

**Comment by jessegreenberg** [comment]
Will be working on this today, should have it closed up by end of day tomorrow but still need to understand exactly what is happening.

**Comment by jessegreenberg** [comment]
This is caused by https://github.com/phetsims/scenery/issues/624 because this sim is using Scenery's DAG feature.

While investigating, I also saw this bit of code in ```CircuitElementNode```
```js
    // Don't enable accessibility for the icons--it causes a bug in the homescreen icon
    if ( options.icon ) {
      delete options.tagName;
    }
```

which is a workaround for this bug, since the icons are used in the screen icons and joist uses the DAG for home screen and navigation bar icons.

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/scenery/issues/624 is closed, and this issue should also be resolved.  I used to be able to reproduce this issue with consistency by adding a wire to the screen and then opening the AboutDialog, and those steps no longer produce this bug.

I also removed these lines
```js
    // Don't enable accessibility for the icons--it causes a bug in the homescreen icon
    if ( options.icon ) {
      delete options.tagName;
    }
```
and the home screen and navigation bar are working. Closing, but @samreid please let me know if you notice other issues.
